lib/messages.h

```diff
@@ -80,7 +81,7 @@ void msg_add_option_group(GOptionContext *ctx);
 
 #define msg_debug(desc, tag1, tags...) 						  \
 	do { 									  \
-	  if (G_UNLIKELY(debug_flag))             				  \
+	  if (G_UNLIKELY(debug_flag) || G_UNLIKELY(startup_debug_flag))             				  \
 	    msg_event_suppress_recursions_and_send(                               \
```

Comment:
I think if we don't use the `startup_debug_flag` in conditions, we won't be able to handle when both `startup_debug_flag` and `debug_flag` are set, so if we run with -dr options, one of the options wont work. If we set the `debug_flag` according to the `startup_debug_flag`, I think we wont be able to tell, whether the `debug_flag` was turned on or off.